static/js/index.js

```
@@ -0,0 +1,859 @@
+                const currentVersion = escapeHtml(update.current_version || "?");
+                const candidateVersion = escapeHtml(update.candidate_version || "?");
+                const source = escapeHtml(update.source || "");
+                const state = String(update.cve_state || "").toLowerCase();
+                const cves = Array.isArray(update.cves) ? update.cves : [];
+
+                const badges = [];
+                if (update.security) badges.push(`<span class="pending-badge pending-badge-security">Security</span>`);
+                if (state === "ready") {
+                    if (cves.length > 0) {
+                        badges.push(`<span class="pending-badge pending-badge-cve">${cves.length} CVE${cves.length > 1 ? "s" : ""}</span>`);
+                        cves.slice(0, 3).forEach(cve => badges.push(`<span class="pending-badge">${escapeHtml(cve)}</span>`));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)